src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -740,10 +752,12 @@ private String getContainersDescriptionForLogging() {
             }
             sb.append("\n");
         }
-        for (String yaml : getYamls()) {
-            sb.append("yaml:\n")
-                .append(yaml)
-                .append("\n");
+        if (getYamls().size() > 0 && isShowRawYaml()) {
```

Comment:
no need to check for `getYamls().size()`

Reply:
Done